fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalCheckPolicy.java

```diff
@@ -230,4 +235,62 @@ public RelatedPolicy(Optional<Expression> rowPolicyFilter, Optional<List<NamedEx
             this.dataMaskProjects = dataMaskProjects;
         }
     }
+
+    private Expression getMaskTypeDef(NereidsParser parser, DataMaskPolicy dataMaskPolicy, Slot slot) {
+        if (dataMaskPolicy instanceof DorisDataMaskPolicy) {
```

Comment:
I don't understand why it's necessary to determine if it's an internal Doris DataMaskPolicy? What is the purpose of this code? If necessary, it should also be placed inside the Doris data mask logic, and there should be no concern about what type of data mask it is

Reply:
Yes, it is better to put it in the data mask logic, but I did it because I don't want to change too much code. I'll put it in the data mask logic and change the code later.